api/controller.go

```diff
@@ -64,9 +64,19 @@ func controllerHandler(w http.ResponseWriter, _ *http.Request) {
 				return
 			}
 
-			err = PostJSON(fmt.Sprintf("%s/start", config.EnvVars.WorkerURL), payloadBytes)
+			resp, err := PostJSON(fmt.Sprintf("%s/start", config.EnvVars.WorkerURL), payloadBytes)
 			if err != nil {
-				log.Error().Err(err).Msgf("error triggering worker for org %s", org)
+				if resp != nil && resp.StatusCode == http.StatusNotAcceptable {
+					log.Warn().Err(err).Msgf("dependabot-circleci not installed on org %s ", org)
+					for _, repo := range repos {
+						err := db.DeleteRepo(repo, context.Background())
+						if err != nil {
+							log.Error().Err(err).Msgf("error deleting repo %s", repo.Repo)
+						}
+					}
```

Comment:
Should we pull this out into a separate function, just to make it easier to read?